src/semgrep_agent/main.py

```diff
@@ -90,32 +94,28 @@ def main(
     maybe_print_debug_info(meta)
     policy = sapp.report_start(meta)
     if sapp.is_configured:
-        if publish_url == "https://semgrep.dev":
-            click.echo(
-                f"| Semgrep Community - logged in as deployment #{sapp.deployment_id}",
-                err=True,
-            )
-        else:
-            click.echo(
-                f"| Semgrep Community - logged in to {publish_url} as deployment #{sapp.deployment_id}",
-                err=True,
-            )
-        if policy:
-            click.echo(f"| policy - using {policy}")
-        else:
-            click.echo(f"| policy - unknown")
+        server = "" if publish_url == "https://semgrep.dev" else f"to {publish_url}"
+        click.echo(
+            get_aligned_command(
+                "manage", f"logged in {server} as deployment #{sapp.deployment_id}"
+            ),
+            err=True,
+        )
+        policy_info = f"using {policy}" if policy else "unknown"
+        click.echo(get_aligned_command("policy", policy_info))
     else:
-        click.echo(f"| Semgrep Community - not logged in", err=True)
+        click.echo(get_aligned_command("manage", f"not logged in"), err=True)
 
     for env_var in [
         "SEMGREP_REPO_URL",
         "SEMGREP_JOB_URL",
         "SEMGREP_BRANCH",
         "SEMGREP_PR_ID",
         "SEMGREP_PR_TITLE",
+        "SEMGREP_REPO_NAME",
```

Comment:
They get printed in this order as well, so could we maybe move this up to be next to REPO_URL?